SparkPost/2web2ui
# SA-1610: Compare By infrastructure

### What Changed
 - Added feature flagged compare by button and drawer

### How To Test
 - Go to an account w/o ui user flag `allow_compare_by` and go to `signals/analytics`. 
 - Verify that there is no compare button.
 - Enable the ui user flag `allow_compare_by` and verify that clicking the compare button opens the drawer. (appteam should be enabled already).
